compiler/erg_compiler/context/initialize/mod.rs

```diff
@@ -44,7 +44,59 @@ use Visibility::*;
 
 const NUM: &str = "Num";
 
-const NAME: &str = "__name__";
+const UNPACK: &str = "Unpack";
+const INHERITABLE_TYPE: &str = "InheritableType";
+const NAMED: &str = "Named";
+const MUTABLE: &str = "Mutable";
+const SELF: &str = "Self";
+const IMMUTIZABLE: &str = "Immutizable";
```

Comment:
nits: should be "Immutable"?

Reply:
This is not a mistake. An `Immutizable` object is an object that can be converted to an `Immutable` object (e.g. `Int!` -> `Int`). It is not itself `Immutable`. However, `Immutizable` is not a common adjective, so I'm trying to think of a better name.

Reply:
Understood. Thank you for the elaboration!